### PR TITLE
Automated tests

### PR DESCRIPTION
**Pull request by Nima-Ra**

### CODE REVIEW

File: automated_tests/.eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...4","25"],{"ruleId":"26","severity":2,"message":"27","line":5,"column":8,"nodeType":"28","endLine":5,"endColumn":18},{"ruleId":"29","severity":2,"message":"30","line":12,"column":20,"nodeType":"31","messageId":"32","endLine":12,"endColumn":43},{"ruleId":"29","severity":2,"message":"30","line":14,"column":19,"nodeType":"31","messageId":"32","endLine":14,"endColumn":32},{"ruleId":"29","severity":2,"message":"30","line":15,"column":19,"nodeType":"31","messageId":"32","endLine":15,"endColumn":32},{"ruleId":"29","severity":2,"message":"30","line":16,"column":24,"nodeType":"31","messageId":"32","endLine":16,"endColumn":37},{"ruleId":"29","severity":2,"message":"30","line":17,"column":25,"nodeType":"31","messageId":"32","endLine":17,"endColumn":38},{"ruleId":"29","severity":2,"message":"30","line":18,"column":18,"nodeType":"31","messageId":"32","endLine":18,"endColumn":31},{"ruleId":"29","severity":2,"message":"30","line":19,"column":20,"nodeType":"31","messageId":"32","endLine":19,"endColumn":42},"import/default","No default export found in imported module \"../idls/profile.did\".","ImportDefaultSpecifier","camelcase","Identifier 'arg_0' is not in camel case.","Identifier","notCamelCase"]
```
Comment (kanej) [inline automated_tests/.eslintcache:1]: Are these permanent exclusions? Or is this a file that should be excluded from git?

**Review by kanej — COMMENTED** [review]
Looks good other than the .eslintcache question

File: automated_tests/.eslintcache (hunk 1 above)
Comment (Nima-Ra) [reply automated_tests/.eslintcache:1]: I should add this file to .gitignore, thanks for reminding me!

**Comment by kanej** [comment]
When I run the tests locally, i get errors. Is there a run guide?

Currently:

```shell
cd automated_tests
yarn
yarn unit:test
```

**Review by kanej — APPROVED** [review]

**Pull request merged**